core/spec.md

```diff
@@ -526,11 +572,70 @@ of the existing entity. Then the existing entity can be deleted.
   all other situations `$structure` MUST NOT be included.
 
 - Constraints:
+  - MUST be unique across all entities in the Registry.
   - MUST be a non-empty absolute URL.
   - MUST be a read-only attribute in API view.
 - Examples:
   - `https://example.com/registry/endpoints/ep1`
 
+##### `shortself` Attribute
+
+- Type: URL
+- Description: A unique absolute URL for an entity. This attribute MUST be
+  an alternative URL for the owning entity's `self` attribute. The intention
+  is that `shortself` SHOULD be shorter in length than `self` such that it
+  MAY be used when the length of the URL referencing the owning entity is
+  important. For example, in cases where the size of a message needs to be
+  as small as possible.
+
+  This specification makes no statement as to how this URL is constructed,
+  to which host/path it references, or whether an HTTP request to this URL
```

Comment:
Possibly remove HTTP from here, to allow for other schemes which may also have redirection?

Reply:
done